### ISSUE TITLE
 Rule description for - ARIA input fields is incorrect

### ISSUE THREAD

**Issue by padmavemulapati** [dequelabs/axe-core#1891]
Steps to reproduce: Verified with below test file, and it has the Violation "ARIA input fields have an accessible name" as a description.
```
<!-- aria-label with empty text string -->
<div id="fail1" aria-label=" " role="combobox">England</div>
 
<!-- The label does not exist. -->
<div id="fail2" aria-labelledby="non-existing" role="combobox">England</div>
 
<!-- The implicit label is not supported on div elements. -->
<label>
    first name
    <div id="fail3" role="textbox"></div>
</label>
 
<!-- explicit label not supported on div elements -->
<label for="fail4">first name</label>
<div role="textbox" id="fail4"></div>
 
<!-- combobox -->
<div id="fail5" role="combobox">England</div>
 
<!-- listbox -->
<div id="fail6" role="listbox" aria-labelledby="label-does-not-exist">
    <div role="option">Orange</div>
</div>
 
<!-- searchbox -->
<div id="fail7"
    role="searchbox"
    contenteditable="true"
    aria-labelledby="unknown-label"></div>
 
 
<!-- slider -->
<div id="fail8"
    role="slider"
    aria-valuemin="1"
    aria-valuemax="7"
    aria-valuenow="2"></div>
 
<!-- spinbutton -->
<div id="fail9"
    role="spinbutton"
    aria-valuemin="0"
    aria-valuemax="10"
    aria-valuenow="8"></div>
 
<!-- textbox -->
<label>foo
    <div id="fail10" role="textbox"></div>
</label>
```
https://dequeuniversity.com/rules/worldspace/3.4/aria-input-field-name

![image](https://user-images.githubusercontent.com/47409579/68660779-d8780e00-055f-11ea-9d52-05946489db27.png)

Expected: ARIA input fields doesn't have an accessible name

**Comment by jeankaplansky** [comment]
The rule description needs to be changed in axe-core itself and then updated in the rule help if the expected description is "ARIA input field does not have an accessible name."

**Comment by padmavemulapati** [comment]
@straker , can you please take a look into this asap

**Comment by dequerags** [comment]
Hello,
Can we get this fixed as soon as possible? Inappropriate description of the issue is confusing & some of the reports that are being sent to client do have this issue. Some subject matter experts are correcting this manually during delivery but not always.

**Comment by straker** [comment]
@dequerags would adding the word "must" fix the issue?

> ARIA input fields must have an accessible name

**Comment by padmavemulapati** [comment]
Verified with the latest axe-expert - seeing "must" in the issues labels. But for the issue
"ARIA input fields have an accessible name" - didnt hold "must" word.
![image](https://user-images.githubusercontent.com/47409579/84029980-e506fc80-a9b0-11ea-8fd8-1786708fbe9e.png)

**Comment by padmavemulapati** [comment]
Seeing the change in the latest code merged.
![image](https://user-images.githubusercontent.com/47409579/84103843-c9900600-aa31-11ea-9dee-053a8891d7df.png)
I will let the SME team once this will be in prod version

**Comment by jeankaplansky** [comment]
rule help update req'd.

**Comment by jeankaplansky** [comment]
PR Submitted: https://github.com/dequelabs/axe-rule-help/pull/47

**Comment by jeankaplansky** [comment]
Fixed in both English and Japanese (title/description only)
